pkg/operator/baremetal_pod.go

```diff
@@ -0,0 +1,645 @@
+                                        "api":     "clusterapi",
+                                        "k8s-app": "controller",
+                                },
+                        },
+                        Template: *template,
+                },
+        }
+}
+
+func newMetal3PodTemplateSpec(config *OperatorConfig) *corev1.PodTemplateSpec {
+        initContainers := newMetal3InitContainers(config)
+        containers := newMetal3Containers(config)
+        tolerations := []corev1.Toleration{
+                {
+                        Key:    "node-role.kubernetes.io/master",
+                        Effect: corev1.TaintEffectNoSchedule,
+                },
+                {
+                        Key:      "CriticalAddonsOnly",
+                        Operator: corev1.TolerationOpExists,
+                },
+                {
+                        Key:               "node.kubernetes.io/not-ready",
+                        Effect:            corev1.TaintEffectNoExecute,
+                        Operator:          corev1.TolerationOpExists,
+                        TolerationSeconds: pointer.Int64Ptr(120),
+                },
+                {
+                        Key:               "node.kubernetes.io/unreachable",
+                        Effect:            corev1.TaintEffectNoExecute,
+                        Operator:          corev1.TolerationOpExists,
+                        TolerationSeconds: pointer.Int64Ptr(120),
+                },
+        }
+
+        return &corev1.PodTemplateSpec{
+                ObjectMeta: metav1.ObjectMeta{
+                        Labels: map[string]string{
+                                "api":     "clusterapi",
+                                "k8s-app": "controller",
+                        },
+                },
+                Spec: corev1.PodSpec{
+                        InitContainers:    initContainers,
+                        Containers:        containers,
+                        PriorityClassName: "system-node-critical",
+                        NodeSelector:      map[string]string{"node-role.kubernetes.io/master": ""},
+                        SecurityContext: &corev1.PodSecurityContext{
+                                RunAsNonRoot: pointer.BoolPtr(false),
+                        },
+                        ServiceAccountName: "metal3-controller",
+                        Tolerations:        tolerations,
+                },
+        }
+}
+
+func getContainerArgs(config *OperatorConfig) []string {
+        args := []string{
+                "--logtostderr=true",
+                "--v=3",
+                fmt.Sprintf("--namespace=%s", config.TargetNamespace),
+        }
+        return args
+}
+
+func newMetal3InitContainers(config *OperatorConfig) []corev1.Container {
+        initContainers := []corev1.Container{
+                {
+                        Name: "ipa-downloader",
+			Image: config.BaremetalPodConfig["BaremetalIpaDownloader"],
```

Comment:
wrong indentation

Reply:
Indentation looks correct in my code but not here though. Not sure why.
Output of go-fmt.sh:
[sdasu@sdasu machine-api-operator]$ sudo ./hack/go-fmt.sh
[sdasu@sdasu machine-api-operator]$ 

[sdasu@sdasu machine-api-operator]$ git diff
[sdasu@sdasu machine-api-operator]$ 

Am I missing something here?

Reply:
I pulled your code and checked.  You've got space based indentation in your changes, while other lines are using tabs.  `hack/go-fmt.sh` didn't work for me either.  I just ran what it did manually and got all the expected fixes:

`find . -name '*.go' ! -path '*/vendor/*' ! -path '*/.build/*' -exec gofmt -s -w {} \+`

Reply:
I was using spaces for indentation. I was able to generate a list of offending lines. Fixing it.

Reply:
@russellb Thanks for giving me that command.  hack/go-fmt.sh kept turning up empty!